packages/babel-transpiler/test/helpers/projectLoader.ts

```diff
@@ -1,8 +1,9 @@
-import { File } from '@stryker-mutator/api/core';
 import * as fs from 'fs';
-import * as glob from 'glob';
 import * as path from 'path';
 
+import { File } from '@stryker-mutator/api/core';
+import * as glob from 'glob';
```

Comment:
I don't really get this formatting.. why is glob here while path and fs are in a different batch?

Reply:
Glob is external package while path and fs are "native node" packages